docs-2.0/graph-computing/algorithm-description.md

```diff
@@ -59,14 +76,27 @@ Parameter descriptions are as follows:
 
 - Nebula Analytics
 
-  |Parameter|Predefined value|Description|
-  |:--|:--|:--|
-  |`TYPE`|`vertex`| The calculation type. Available values are `vertex` and `subgraph`. When set to `vertex`, the system calculates the number of cores for each vertex.|
-  |`VERTICES`|`0`| The number of vertexes. If set to `0`, the system automatically calculates the value.|
-  |`EDGES`|`0`| The number of edges. If set to `0`, the system automatically calculates the value.|
-  |`KMIN`|`1`| Set the minimum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`. |
-  |`KMAX`|`1000000`| Set the maximum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`.|
-  |`ITERATIONS`|`10`| The maximum number of iterations.|
+  - Input parameters
+
+    |Parameter|Predefined value|Description|
+    |:--|:--|:--|
+    |`TYPE`|`vertex`| The calculation type. Available values are `vertex` and `subgraph`. When set to `vertex`, the system calculates the number of cores for each vertex.|
+    |`KMIN`|`1`| Set the minimum value of K when range calculation. Takes effect only when `TYPE`=`subgraph`. |
```

Comment:
what do you mean by when range calculation? does it refer to when calculating the range?